requirements.txt

```diff
@@ -1,6 +1,7 @@
 k2
 kaldialign
 lhotse@git+https://github.com/lhotse-speech/lhotse
+prettytable
```

Comment:
not sure if we really need to make this as an requirement, it seems to me that it's not so necessary?

Reply:
Will remove it. I added it in case someone wants to reproduce the result.